public/chassis.css

```diff
@@ -0,0 +1,392 @@
+@import url("https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;600;700;800&display=swap");
+

```

Comment:
It is not a good practice to use import inside CSS files because it will wait for the import to complete and code below the import won't get load, so it may create issues.

Correct me If I am wrong.

Reply:
Yes, you are right. Thanks for pointing this out. I have linked the font on the html page as suggested.